README.md

```diff
@@ -41,5 +39,5 @@ Or use QEMU. First, you need to install it:
 Then to run the kernel, just type:
 ```shell
 # You may need to change qemu-system-aarch64 by whatever is QEMU for aarch64 is named on your computer.
-qemu-system-aarch64 -M raspi3b -serial stdio -kernel build/kernel8.elf
+qemu-system-aarch64 -M raspi3b -serial stdio -kernel build/kernel/kernel8.bin
```

Comment:
We should keep .elf as QEMU support the ELF format. This avoid the extra objcopy step and probably also keep debug information if we connect gdb to QEMU.

Let only use .bin when flashing the kernel into the real hardware.